### PR TITLE
Added contributing.md file

### PR DESCRIPTION
**Pull request by Raffo**
@szuecs Please review.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Please open an issue for bugs and feature requests. We are open to Pull Requests, but we would like to discuss features with you.
+If you want to submit a PR, always use feature branches and let people discuss changes in pull requests.
+
+Pull requests should only be merged after all discussions have been concluded and at least 2 reviewers has given their
+**approval**.
+
+## Guidelines
+
+- **every code change** needs a test
```
Comment (szuecs) [inline CONTRIBUTING.md]: We can not be so strict, if we do not provide many tests ourselves.

Comment (Raffo) [inline CONTRIBUTING.md]: To me it means: no more PRs with no tests, us included. Wouldn't this make sense?

Comment (szuecs) [inline CONTRIBUTING.md]: I don't like to be so strict, so instead of "needs" having a "should" would be ok for me

**Comment by Raffo** [comment]
Agreed.

**Comment by szuecs** [comment]
:+1:

**Comment by astarnell** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**